WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Split Shipments/MoveToShipmentNotice.swift

```diff
@@ -82,8 +83,8 @@ private extension MoveToShipmentNotice {
     }
 }
 
-private extension MoveToShipmentNotice {
-    enum Layout {
+extension MoveToShipmentNotice {
+    private enum Layout {
```

Comment:
Exposing the `Localization` enum to reuse the localized title of the Move to new shipment button title.